packages/main/src/webComponents/Table/Table.mdx

```diff
@@ -58,27 +67,83 @@ const GrowingTable = () => {
   };
   return (
     <div style={{ height: '250px', overflow: 'auto' }}>
-      <Table
-        onLoadMore={onLoadMore}
-        growing={TableGrowingMode.Scroll}
-        headerRow={
-          <TableHeaderRow>
-            <TableHeaderCell>
-              <Label>Column 1</Label>
-            </TableHeaderCell>
-            <TableHeaderCell>
-              <Label>Column 2</Label>
-            </TableHeaderCell>
-          </>
-        }
-      >
+      <Table {...args} features={<TableGrowing onLoadMore={onLoadMore} type={TableGrowingMode.Scroll} />}>
```

Comment:
I would add the `headerRow` in this example for the sake of completeness. Users who a not familiar with storybook might be confused what `...args` should be

Reply:
I've added the rows and used `{...otherProps}` so it's clear that the table does not only implement the given props. E.g. if they're changed via Storybook.